### ISSUE TITLE
Error when sending SMS from USA number to international mobile

### ISSUE THREAD

**Issue by david-nedved** [mvaragnat/talkative-twilio#1]
Hi as the title suggests I'm seeing the following error when trying to send a SMS... Twillio account is setup and works fine when manually making API request via CURL...

```
Missing error handler on `socket`.
SyntaxError: Unexpected token :
    at Socket.<anonymous> (app/controllers/socket.js:46:20)
```

**Comment by mvaragnat** [comment]
Are things working fine for you with USA to USA numbers ? To be honest I tested it only with France to France so far

**Comment by david-nedved** [comment]
Yes; I can send/rcv messages fine with USA <--> USA numbers and also USA <--> AU numbers when CURL'ing the Twilio API directly
